test/controllers/EditSetControllerSpec.scala

```diff
@@ -69,846 +66,89 @@ class EditSetControllerSpec extends BaseSpec {
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (C)",
-                coveringDates = "1964"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/4",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (D)",
-                coveringDates = "1961"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/5",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (E)",
-                coveringDates = "1963"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/6",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (F)",
-                coveringDates = "1965"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/7",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (G)",
-                coveringDates = "1967"
-              )
-            ),
-            numberOfPages = 2
-          )
-        )
+        verify(mockEditSetService).getEditSet(anyString())
 
+        editSet
       }
 
-      "Page 2 sorted by CCR, ascending" in {
-        val page = orderingRequest("ccr", "ascending", offset = 2)
+      "with request parameters" in {
+        val page = orderingRequest("ccr", "ascending")
 
         status(page) mustBe OK
-        assertPageAsExpected(
-          asDocument(page),
-          ExpectedEditSetPage(
-            title = "Browse Edit Set (Page 2 of 2)",
-            caption = "Showing 11 - 12 of 12 records",
-            header = "Edit set: COAL 80 Sample",
-            ccrTableHeader = ExpectedTableHeader("ccr", "descending"),
-            scopeAndContentTableHeader = ExpectedTableHeader("scope-and-content", "ascending"),
-            coveringDateTableHeader = ExpectedTableHeader("covering-dates", "ascending"),
-            expectedSummaryRows = Seq(
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/8",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (H)",
-                coveringDates = "1969"
-              ),
-              ExpectedEditSetSummaryRow(
-                ccr = "COAL 80/80/9",
-                scopeAndContents =
-                  "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (I)",
-                coveringDates = "1971"
-              )
-            ),
-            numberOfPages = 2
-          )
-        )
-
+        contentType(page) mustBe Some("text/html")
```

Comment:
Why have the content assertions been removed?

Reply:
As above.

Reply:
As @rwalpole wrote above, I agree that the content type assertions should be removed.

Reply:
Removed

Reply:
Confirmed.